src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import Operate from './operate';
+
+const Calculate = ({ total, next, operation }, buttonName) => {
+  const data = { total, next, operation };
+
+  if (buttonName === 'AC') {
```

Comment:
- [ ] What if the button is '='? What would happen to the value of `next` and `operation` then?
- [ ] What if the button is `+/-`?